source/priority_queue.c

```diff
@@ -145,7 +158,11 @@ int aws_priority_queue_init_dynamic(
     queue->pred = pred;
     AWS_ZERO_STRUCT(queue->backpointers);
 
-    return aws_array_list_init_dynamic(&queue->container, alloc, default_size, item_size);
+    bool ret = aws_array_list_init_dynamic(&queue->container, alloc, default_size, item_size);
```

Comment:
It's minor but I would prefer ret to be an error int  (and compare against AWS_OP_SUCCESS) rather than converting to bool and implicitly go back and forth.  It's easy to mentally assume a bool is "success" but here it's the opposite.

Reply:
Resolved, could you merge @bretambrose